Watchman.Discord/Areas/Protection/Models/ServerSafeUsers.cs

```diff
@@ -0,0 +1,58 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using Watchman.DomainModel.Messages;
+
+namespace Watchman.Discord.Areas.Protection.Models
+{
+    public class ServerSafeUsers
+    {
+        public ulong ServerId { get; }
+        public List<ulong> SafeUsers { get; private set; }
```

Comment:
czy ta lista może być edytowana z zewnątrz?
jeśli nie - a domyślam sie że nie, bo metoda IsUserSafe() jest prywatna 

to lepiej by było, gdyby tutaj był hashset -> wyszukiwanie będzie szybsze 
albo tablice 

listy są bardzo słabe pod kątem wyszukiwania danych 
ich główny plus to szybkość edycji i szybkość dodawania kolejnych elementów